guide/05-working-with-the-spatially-enabled-dataframe/part2_data_io_reading_data.ipynb

```diff
@@ -0,0 +1,5229 @@
+{
```

Comment:
<p>The <a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html#geoaccessor" target="_blank"><strong>Spatially enabled DataFrame</strong></a> can read from many <strong>sources</strong>, including <a href="https://doc.arcgis.com/en/arcgis-online/share-maps/hosted-web-layers.htm" target="_blank">Feature layers</a>, <a href="http://desktop.arcgis.com/en/arcmap/latest/manage-data/feature-classes/a-quick-tour-of-feature-classes.htm" target="_blank">Feature classes</a>, <a href="http://desktop.arcgis.com/en/arcmap/latest/manage-data/shapefiles/what-is-a-shapefile.htm" target="_blank">Shapefiles</a>, Pandas <a href="http://pandas.pydata.org/pandas-docs/stable/dsintro.html#dataframe" target="_blank">DataFrames</a> and more. Let's dive into the details of accessing GIS data from various sources.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>